src/lib/util/calcMaxTripLength.ts

```diff
@@ -44,6 +44,10 @@ export function calcMaxTripLength(user: MUser, activity?: activity_type_enum) {
 			max *= 2;
 			break;
 		}
+		case 'NightmareZone': {
+			max *= 3;
+			break;
+		}
```

Comment:
I'm thinking 2 or 2.5.

Reply:
![temp](https://user-images.githubusercontent.com/68188574/212924960-2d31243a-51a7-465d-8fd1-7fdfa8309802.PNG)

Also if it's less than 3x, NMZ will be dead content on release since normal combat training methods in bot beat it. The only benefit with NMZ will be to send 1x trip each day before bed time.